pkg/operator/defaultstorageclass/controller.go

```diff
@@ -98,10 +98,13 @@ func (c *Controller) sync(ctx context.Context, syncCtx factory.SyncContext) erro
 				Reason:  "UnsupportedPlatform",
 				Message: syncErr.Error(),
 			}
+			availableCnd.Message = "No default StorageClass for this platform"
```

Comment:
Could you add a brief comment saying why we set it to true (like the above)?

After reading the message, one may assume the condition is wrong

Reply:
A comment added, PTAL